src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -298,8 +298,8 @@ private JsonNode queryAggregatedObject(String id) throws URISyntaxException, IOE
         //The response contains the aggregated object as a jsonstring. Makes it this wierd to get out.
         ResponseEntity<String> response = httpRequest.performRequest();
         JsonNode body =  objectMapper.readTree(response.getBody());
-        JsonNode responseEntity = objectMapper.readTree(body.get("responseEntity").asText());
-        actualJSON = responseEntity.get(0);
+        JsonNode responseEntity = body.get("queryResponseEntity");
+        actualJSON = responseEntity;
```

Comment:
I think you can remove the "actualJSON" here and return responseEntity directly. It's just unnecessary now after your fix.

Reply:
Well, later in the other methods we need to get the "queryResponseEntity" for comparison.... or we may need to add the "queryResponseEntity " string in the expected json

Reply:
I think I misunderstood.... I removed the "JsonNode actualJSON = null;"